File: PrimeMinistersByJava/primeministers/Tuple.java
package primeministers;

import java.util.List;

public class Tuple {
    private Attributes attributes;
    private List<String> values;

    public Tuple(final List<String> valueCollection) {
		this.values = valueCollection;
        return;
    }

    public Attributes attributes() {
        return this.attributes;
    }

	public void add(int index, String aString) {
        this.values().add(index, aString);
    }

    public List<String> values() {
        return this.values;
    }
}

File: PrimeMinistersByPython/primeministers/tuple.py
class Tuple:
	def __init__(self, value_collection):
		self.values = value_collection
		return

	# def get_attributes(self):
	# 	return self.attributes

	def add(self, an_index, a_string):
		self.get_values().add(an_index, a_string)

	def get_values_with_index(self, an_index):
		return self.get_values()[an_index]

	def get_values(self):
		return self.values
	
	def set_values(self, values):
		self.values = values